README.md

```diff
@@ -312,8 +312,9 @@ def marshal_dump; end
 The following example implements an `ActiveRecord` store to save exchange rates to a database.
 
 ```ruby
-# DB columns :from[String], :to[String], :rate[Float]
+# rails g model exchange_rate from:string to:string rate:float
 
+# for Rails 5 replace ActiveRecord::Base to ApplicationRecord
```

Comment:
`with` instead of `to`

Reply:
Thanks @antstorm I just amended the commit